_posts/en/pages/2017-01-01-download.md

```diff
@@ -56,11 +56,15 @@ releasekeys: "Bitcoin Core Release Signing Keys"
 
 pgp_key_fingerprint: "PGP key fingerprint"
 verify_download: "Verify your download"
-verification_recommended: "Download verification is optional but highly recommended.  Click one of the lines below to view verification instructions for that platform."
+verification_recommended: >
+  Download verification is optional but highly recommended.
+  Click one of the lines below to view verification instructions for that platform.
+  Note that due to a <a href=\"https://bugs.launchpad.net/snapstore/+bug/1803914\">series</a>
+  <a href=\"https://bugs.launchpad.net/launchpad/+bug/1331914\">of</a> mishandlings of security issues
+  by Canonical/Ubuntu, use of Bitcoin wallets on Ubuntu is strongly discouraged.
 windows_instructions: "Windows verification instructions"
 macos_instructions: "MacOS verification instructions"
 linux_instructions: "Linux verification instructions (not for Ubuntu PPA)"
```

Comment:
Mentioning ubuntu PPA on this line is probably more confusing than not. Would remove this too.

Reply:
Oops, good point.